cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/utils/NGSIUtils.java

```diff
@@ -60,7 +60,11 @@ private NGSIUtils() {
      */
     public static String encode(String in, boolean deleteSlash, boolean encodeSlash) {
         if (deleteSlash) {
-            return ENCODEPATTERN.matcher(in.substring(1)).replaceAll("_");
+           if(in.startsWith("/")) {
+                return ENCODEPATTERN.matcher(in.substring(1)).replaceAll("_");
+            }else {
+                return ENCODEPATTERN.matcher(in).replaceAll("_");
+            }
```

Comment:
Indent looks weird... Maybe better:

```suggestion
            if(in.startsWith("/")) {
                return ENCODEPATTERN.matcher(in.substring(1)).replaceAll("_");
            } else {
                return ENCODEPATTERN.matcher(in).replaceAll("_");
            }
```

Reply:
Resolved in b539526